src/main/java/com/sublinks/sublinksapi/slurfilter/services/SlurFilterService.java

```diff
@@ -50,7 +50,7 @@ public SlurFilter getHighestSlurFilterMatchingText(String text) {
       return null;
     }
 
-    return slurFilters.stream().filter((slur) -> text.matches(slur.getSlurRegex()))
```

Comment:
String.matches() only returns true when the _entire_ string matches. Opted to just fix it here vs creating extra leg work with another issue and juggling tests.